audio_processing.py

```diff
@@ -57,19 +59,26 @@ def process(recording, api, s3):
 
     with tempfile.TemporaryDirectory() as temp:
         temp_path = Path(temp)
-        input_filename = temp_path / ("recording." + input_extension)
-
+        input_filename = temp_path / ("recording" + input_extension)
+        wav_filename = temp_path / "output.wav"
         logging.info("downloading recording to %s", input_filename)
         s3.download(recording["rawFileKey"], str(input_filename))
-
-        output_filename, new_mime_type = encode_file(input_filename)
-
+        data, sr = librosa.core.load(str(input_filename), sr=None)
+        amplification = normalize(data, MAX_AMPLIFICATION)
+        newMetadata["additionalMetadata"] = { "amplification": amplification }
+        librosa.output.write_wav(str(wav_filename), data, sr)
+        output_filename, new_mime_type = encode_file(wav_filename)
         logging.info("uploading from %s", output_filename)
         new_key = s3.upload(str(output_filename))
 
-    api.report_done(recording, new_key, new_mime_type)
+    api.report_done(recording, new_key, new_mime_type, newMetadata)
     logging.info("Finished processing")
 
+def normalize(data, maxAmp):
```

Comment:
The most widely adopted naming style for variables in Python code is `lower_case_with_underscores`, so `maxAmp` would be better as `max_amp`.